primefaces/src/main/java/org/primefaces/component/autocomplete/AutoCompleteRenderer.java

```diff
@@ -506,7 +508,10 @@ protected void encodeSuggestions(FacesContext context, AutoComplete ac, Object i
         Converter converter = ComponentUtils.getConverter(context, ac);
 
         if (customContent) {
-            encodeSuggestionsAsTable(context, ac, items, converter);
+            FailableRunnable<IOException> runnable = () -> {
```

Comment:
instead of declaring it, I don't mind if it's inline

Reply:
Inlined...